crates/viewer/re_view_spatial/src/contexts/transform_tree_context.rs

```diff
@@ -473,20 +502,22 @@ fn transform_info_for_downward_propagation(
             reference_from_pinhole_entity: reference_from_entity,
         });
         reference_from_entity *= entity_from_2d_pinhole_content;
-
-        // Need to update per instance transforms as well if there are poses!
-        if has_instance_transforms {
-            *reference_from_instances.first_mut() = reference_from_entity;
-        } else {
-            for reference_from_instance in &mut reference_from_instances {
-                *reference_from_instance *= entity_from_2d_pinhole_content;
-            }
-        }
```

Comment:
full disclosure: I have no idea why this re-transform of the instance poses was there when the actual collection happened earlier already. Looks 100% wrong to me now. I bet we just never ran into this because nobody reprojects things with instance poses

begs the question why there's no test. Should add one if it's not too much work :/